experimental/BasisLieHighestWeight/src/BirationalSequence.jl

```diff
@@ -1,27 +1,28 @@
 struct BirationalSequence
   operator_roots::Vector{RootSpaceElem}
   operator_weights::Vector{WeightLatticeElem}
+  root_system::RootSystem
 
   function BirationalSequence(
-    operator_roots::Vector{RootSpaceElem}, operator_weights::Vector{WeightLatticeElem}
+    operator_roots::Vector{RootSpaceElem}, operator_weights::Vector{WeightLatticeElem}, root_system::RootSystem
   )
     @req length(operator_roots) == length(operator_weights) "Different lengths"
```

Comment:
This constructor should check that all roots and weights relate to the correct root system